include/ctype.h

```diff
@@ -22,66 +22,93 @@ extern "C" {
 #endif
 
 
-/* This function checks whether the passed character is alphanumeric. */
-extern int isalnum(int c);
+/* This function checks whether the passed character is lowercase letter. */
+int islower(int c);
+#define _islower(c) (((c) >= 'a' && (c) <= 'z') ? 1 : 0)
```

Comment:
Small C nitpick.
C99 7.1.3 p 1 states:
— All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.
— All identifiers that begin with an underscore are always reserved for use as identifiers with file scope in both the ordinary and tag name spaces.

C99 6.2.3 states that macros is not considered as identifiers in terms of scope and namespaces.

If my understanding of those is correct, then macros in headers should begin with __ (double _).

Reply:
Done